src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
+import java.util.Locale;
```

Comment:
Nothing to say about formatting of any kind (or at least that I've seen), pretty much perfect.